### DISCUSSION TITLE
New Private repository for review: Context Mapping Quiz

### DISCUSSION THREAD

**Discussion by NTCoding** [ddd-crew/ddd-starter-modelling-process#56] [Team Posts]
Hey everyone.

@selketjah and myself have been discussing the idea of sharing the context mapping quiz we designed for our DDD workshops. I've finally put together a private repository here: https://github.com/ddd-crew/context-mapping-quiz/

All feedback is appreciated. 

Thanks

**Comment by yellowbrickc** [comment]
Sorry, I have overseen this completely (just like the other 3 messages in this group 🙁 )

I can imagine that the quiz works very good between 2 training days or to work alone during the training. But this is because you can evaluate the result and can give answers. How could this work asynchronously, in a repo? Or do you just want to share to be used in other workshops?

**Comment by NTCoding** [comment]
Hey @yellowbrickc,

The intended usage for this quiz is to be used as content in a workshop. People can copy the contents of the Miro board into their own Miro board.
